platforms/jvm/plugins-application/src/test/groovy/org/gradle/api/plugins/ApplicationPluginTest.groovy

```diff
@@ -30,25 +29,6 @@ import org.gradle.util.TestUtil
 class ApplicationPluginTest extends AbstractProjectBuilderSpec {
     private final ApplicationPlugin plugin = TestUtil.newInstance(ApplicationPlugin)
 
-    def "applies JavaPlugin and adds convention object with default values"() {
```

Comment:
Part of this test verifies the extension's default values. Can we leave that part in?